azurerm/resource_arm_recovery_services_protected_vm_test.go

```diff
@@ -331,6 +379,209 @@ resource "azurerm_recovery_services_protected_vm" "test" {
 `, testAccAzureRMRecoveryServicesProtectedVm_base(rInt, location))
 }
 
+// For update backup policy id test
+func testAccAzureRMRecoveryServicesProtectedVm_basePolicyTest(rInt int, location string) string {
+	rstr := strconv.Itoa(rInt)
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%[1]d"
+  location = "%[2]s"
+}
+
+resource "azurerm_virtual_network" "test" {
+  name                = "vnet"
+  location            = "${azurerm_resource_group.test.location}"
+  address_space       = ["10.0.0.0/16"]
+  resource_group_name = "${azurerm_resource_group.test.name}"
+}
+
+resource "azurerm_subnet" "test" {
+  name                 = "acctest_subnet"
+  virtual_network_name = "${azurerm_virtual_network.test.name}"
+  resource_group_name  = "${azurerm_resource_group.test.name}"
+  address_prefix       = "10.0.10.0/24"
+}
+
+resource "azurerm_network_interface" "test" {
+  name                = "acctest_nic"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+
+  ip_configuration {
+    name                          = "acctestipconfig"
+    subnet_id                     = "${azurerm_subnet.test.id}"
+    private_ip_address_allocation = "Dynamic"
+    public_ip_address_id          = "${azurerm_public_ip.test.id}"
+  }
+}
+
+resource "azurerm_public_ip" "test" {
+  name                = "acctest-ip"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  allocation_method   = "Dynamic"
+  domain_name_label   = "acctestip%[1]d"
+}
+
+resource "azurerm_storage_account" "test" {
+  name                     = "acctest%[3]s"
+  location                 = "${azurerm_resource_group.test.location}"
+  resource_group_name      = "${azurerm_resource_group.test.name}"
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
+}
+
+resource "azurerm_managed_disk" "test" {
+  name                 = "acctest-datadisk"
+  location             = "${azurerm_resource_group.test.location}"
+  resource_group_name  = "${azurerm_resource_group.test.name}"
+  storage_account_type = "Standard_LRS"
+  create_option        = "Empty"
+  disk_size_gb         = "1023"
+}
+`, rInt, location, rstr[len(rstr)-5:])
+}
+
+// For update backup policy id test
+func testAccAzureRMRecoveryServicesProtectedVm_withVault(rInt int, location string) string {
+	return fmt.Sprintf(`
+%[1]s
+
+resource "azurerm_recovery_services_vault" "test" {
+		name                = "acctest-%[2]d"
```

Comment:
Could we use spaces instead of tabs here for consistency?

Reply:
Converted to space